src/App.vue

```diff
@@ -220,7 +209,6 @@ export default class App extends Mixins(DateMixin, FilingTemplateMixin, LegalApi
 
     return this.getNRAuthorizations(queryNrNumber).then(async data => {
       this.storeAuthRoles(data) // throws if no role
-      await this.intializeNameXToken() // TODO : temporary token retrieval code
       const nrResponse = await this.queryNameRequest(queryNrNumber)
 
       // NR not found
```

Comment:
In Business Filings UI, I added some extra validity checks that you might also want to implement.
See https://github.com/bcgov/business-filings-ui/blob/cb421eed27d97bc63dc99596d42277d3be7e0726/src/App.vue#L422

Reply:
Thanks! I added that function to the name request mixin and added a state to the NR invalid dialog as well as updated the tests.